.travis.yml

```diff
@@ -19,8 +19,9 @@ install:
   - pip install pytest-django
   - pip install pytest-xdist
   - export SECRET_KEY="abcde%!z@cu@o29$zjifeoudt1ute&0ii33!8huud71&4#l2#817pei"
-  - git clone https://github.com/open-source-economics/B-Tax
-  - cd B-Tax && python setup.py develop && export BTAX_CUR_DIR=`pwd`/btax && cd ..
+  - git clone https://github.com/PeterDSteinberg/B-Tax
+  - cd B-Tax && git fetch --all && git checkout psteinberg/fix-groupings
```

Comment:
Correct.  Just until we merge that PR in B-Tax that adds `DEFAULT_ROW_GROUPINGS` global var.